src/quickstart/smart-contract.md

```diff
@@ -98,7 +113,7 @@ Next, we'll define a storage value. In our case, we have a single counter that w
 
 An ABI defines an interface, and there is no function body in the ABI. A contract must either define or import an ABI declaration and implement it. It is considered best practice to define your ABI in a separate library and import it into your contract because this allows callers of the contract to import and use the ABI in scripts to call your contract.
```

Comment:
I think we can reword this a little: "An ABI defines an interface for a contract. A contract must either define or import an ABI declaration. It is considered best practice to define your ABI in a separate library and import it into your contract because this allows callers of the contract to import and use the ABI in scripts to call your contract."